sdk/src/system_impl.cpp

```diff
@@ -104,7 +104,11 @@ buildCamera(std::unique_ptr<SensorEnumeratorInterface> enumerator) {
     return camera;
 }
 
-SystemImpl::SystemImpl() {}
+SystemImpl::SystemImpl() {
+    LOG(INFO) << "SDK version: " << (std::string)VERSION
```

Comment:
Make sure this prints only once no matter how many instances of SystemImpl we create.